src/main/java/org/bananarama/crud/DeleteOperation.java

```diff
@@ -53,4 +53,11 @@
      * @return
      */
     public DeleteOperation<T> from(Stream<T> data,QueryOptions options);
+    
+    /**
+     * Delete all records from the persisted layer.
+     * @param data The collection of objects to be removed
+     * @return
+     */
+    public DeleteOperation<T> all();
```

Comment:
:confused: I don't know. I want to break the compatibility with the other adapters.
But if we want to guarantee the backward compatibility, I would prefer to throw an exception:

`    default public DeleteOperation<T> all(){
        throw new UnsupportedOperationException("Not supported yet.");
    }`